pkg/planner/core/logical_plan_builder.go

```diff
@@ -7288,10 +7288,10 @@ func collectTableName(node ast.ResultSetNode, updatableName *map[string]bool, in
 	}
 }
 
-func appendDynamicVisitInfo(vi []visitInfo, priv string, withGrant bool, err error) []visitInfo {
+func appendDynamicVisitInfo(vi []visitInfo, privs []string, withGrant bool, err error) []visitInfo {
 	return append(vi, visitInfo{
 		privilege:        mysql.ExtendedPriv,
-		dynamicPriv:      priv,
+		dynamicPrivs:     privs,
```

Comment:
Is this change necessary? Maybe you can 'git checkout master xxx.go' for most of the changes. @glorv

Reply:
Yes, I think this change is necessary to express that user should have at least one of the privileges. I added a comment for the field `dynamicPrivs`